redis/connection.py

```diff
@@ -1044,6 +1044,12 @@ def __repr__(self):
             repr(self.connection_class(**self.connection_kwargs)),
         )
 
+    def __eq__(self, other):
+        return (
```

Comment:
Do you think it's better written as-is, or using duck typing as so?

    try:
        return self.connection_kwargs == other.connection_kwargs
    except AttributeError:
        return False

Now that I think about it, I'm leaning towards duck typing...